common/djangoapps/student/helpers.py

```diff
@@ -521,47 +533,40 @@ def _cert_info(user, course_overview, cert_status):
     if status != default_status and course_overview.end_of_course_survey_url is not None:
         status_dict.update({
             'show_survey_button': True,
-            'survey_url': process_survey_link(course_overview.end_of_course_survey_url, user)})
+            'survey_url': process_survey_link(course_overview.end_of_course_survey_url, user),
+        })
     else:
         status_dict['show_survey_button'] = False
 
     if status == 'downloadable':
+        linkedin_config = LinkedInAddToProfileConfiguration.current()
+        share_settings = configuration_helpers.get_value("SOCIAL_SHARING_SETTINGS", settings.SOCIAL_SHARING_SETTINGS)
```

Comment:
single quotes :)

Reply:
I fix so many spots and rather than congratulate on those, I get this...

My bad on the miss. Will fix. Copy Pasta mistake.